core/src/test/java/cucumber/runtime/formatter/PrettyFormatterTest.java

```diff
@@ -381,6 +382,37 @@ public void should_mark_arguments_in_steps() throws Throwable {
                                           AnsiEscapes.GREEN + AnsiEscapes.INTENSITY_BOLD + "arg2"  + AnsiEscapes.RESET));
     }
 
+    // Reproduce issue #619: This test should now fail!
+    @Test(expected = StringIndexOutOfBoundsException.class)
+    public void nested_capture_groups_cannot_be_handled(){
```

Comment:
Will remove this test. I only kept it as the "guidelines for contributing" mentioned something about having a test to reproduce a bug...